installer

```diff
@@ -0,0 +1,139 @@
+#!/usr/local/cpython-3.4/bin/python3
+
+"""
+Copy modules from here to --prefix.
+
+Note that we do not scan for directories; instead we have a hardcoded list in all_directories.
```

Comment:
Why?

Reply:
No particular reason.  One way you could accidentally copy a new doc/, the other you could accidentally not copy a new from-github.